src/main/kotlin/data/Collection.kt

```diff
@@ -0,0 +1,10 @@
+package data
+
+data class Collection(
+        var id: Int,
+        var title: String,
+        var sort: Int,
+        var label: String,
+        var srcFile:String,
```

Comment:
what?

Reply:
so for collection we thought that we would store the title of the collection, the sort if it has an ordering, a label to say what it is, and any source files it may have. However if we are breaking collections down to anthology, books, chapters and such, then perhaps this class doesn't need to exist?